### PR TITLE
1.14.8 Release Notes

### PR DESCRIPTION
**Pull request by chromatic**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -21,276 +20,93 @@ Compatibility
...
-The user can select which stats are returned to save processing time for stats
-that aren't wanted. For more information, use:
+Previously, Dogecoin Core nodes would maintain a large list of inventory for
+each peer. This is excessive and unnecessary and has been reduced to a more
+reasonable number.
 
-```
-dogecoin-cli help getblockstats
-```
+*Implemented with #...*
```
Comment (patricklodder) [inline doc/release-notes.md:39]: Missing PR number: `#3577`

File: doc/release-notes.md [hunk 2]
```diff
@@ -21,276 +20,93 @@ Compatibility
...
-*Implemented with #3297*
+### Improve Transaction Request Tracking
 
+A new transaction request tracking system improves the way Dogecoin Core
+communicates with other peers. Notably, this improvement prioritizes
+communication with outbound peers over inbound peers and makes CPU and memory
+use more reliable and scalable with the number of peers.
 
-### Add a `height` parameter to `-walletnotify`
+*Implemented with #...*
```
Comment (patricklodder) [inline doc/release-notes.md:48]: Same PR as above: `#3577`

**Review by patricklodder — COMMENTED** [review]
Text is good. The reference to 1 PR is missing (see inline comments)

Other than that, remaining reviewers need to be added to the list of contributors:

- daanksy
- Dakoda Greaves
- Michi Lumin

When that's done, I think we're good with this.

**Comment by chromatic** [comment]
I think the only possible other note item we might be missing is #3611 , which isn't merged yet.

**Comment by patricklodder** [comment]
> might be missing is #3611 , which isn't merged yet.

Merged it.

**Comment by chromatic** [comment]
I added a note about `SECURITY.md`. This is ready for review. I'll squash after a thumbs-up before final acknowledgment.

**Review by edtubbs — APPROVED** [review]
ACK, there may be more from [#3587](https://github.com/dogecoin/dogecoin/issues/3587#issue-2421533237) but the notes look good.

**Review by patricklodder — APPROVED** [review]
ACK - this is complete for the release too.

**Pull request merged**